docs/webhooks/08-Certificates.md

```diff
@@ -28,4 +28,40 @@ It is our recommendation that customers configure their servers to trust the roo
 
 ## New Certificates
 
-PagerDuty Webhook certificates are rotated yearly.  The new certificate(s) will be made available on this page in the November 2021 - December 2021 time frame.
+PagerDuty webhook certificates are rotated yearly. The next certificate rotation will occur on January 5th, 2022 for both the US and EU service regions. The new certificates are provided below.
+
+### US Service Region (webhooks.pagerduty.com)
+
+#### PagerDuty Webhooks Certificate x.509 ([download](https://developer.pagerduty.com/certificates/2022_webhooks_pagerduty_com.pem))
```

Comment:
Nit: Is it worth putting the download link below the header so it is easily distinguishable as a link?

Reply:
We should do that but i'm having trouble previewing the changes with the current Stoplight service degradation.  Maybe we can do it as a follow-up change...